lib/src/node.dart

```diff
@@ -34,10 +36,10 @@ void main() {
 /// possible.
 ///
 /// [render]: https://github.com/sass/node-sass#options
-void _render(
-    NodeOptions options, void callback(NodeError error, NodeResult result)) {
+Future _render(NodeOptions options,
+    void callback(NodeError error, NodeResult result)) async {
```

Comment:
Why are you changing the types here? Node doesn't understand what a "Future" is, so this will definitely break it.